SER-48 Drop bootstrap

sifbiri opened the pull request:
## Purpose

Drop dependency on bootstrap for styling.

## Related Issues

Closes https://sig-gis.atlassian.net/browse/SER-48

## Submission Checklist

- [x] Included Jira issue in the PR title (e.g. `SER-### Did something here`)
- [x] Code passes linter rules for each file you updated. To lint all files at once, run `npm run lint`. To just lint one specific file run `npx quick-lint-js src/js/<file-you-changed> --snarky`.
- [x] No new reflection warnings (`clojure -M:check-reflection`)

## Testing

The UI should still look and feel similar and with no issues.

Oliver-BE commented on src/js/components/NICFIControl.jsx:
```diff
@@ -59,16 +59,15 @@ export default function NICFIControl({ extraParams, setParams, nicfiLayers }) {
-      <Button
-        className="mt-2"
-        onClick={() =>
+      <Button style={{ marginTop: "0.5rem" }}>
+        onClick=
+        {() =>
```
It looks like the `onClick` function is outside of the `<Button>` element.

Oliver-BE reviewed (commented):
I noticed a few small things visually. For the below reference photo, your branch is on the left side and the dev site is on the right side.

* The Login button needs a bit of padding 
* The text for the "Update NICFI" layer isn't right
![Screenshot from 2022-08-15 17-07-55](https://user-images.githubusercontent.com/40574170/184756744-2a38064b-e4fd-411e-9b68-e7aa47674dec.png)

Also, I have a question about a few of the files inside of `resources/public/css`. It looks like we have a `bootrstrap-grid` CSS file as well as a few `tabulator` CSS files. Which files of those (if any) do we still need to keep around? Also, what does the "gmw" in `gmw_global.css` stand for? What's the difference between `gmw_global.css` and `gmw_main.css`?

sifbiri commented:
I have fixed padding and NICFI button. We don't need to sweat the details to be pixel perfect comparing to previous one. We will style it all over again using Andrea designs.

The other css files `like bootstrap-grid` and `tabulator` were not being served but just sitting in `resources` directory, I have removed them. `gmw_global.css` apply styles to html elements (reseting the styles) and `gmw_main.css` defines style classes that get used by those html elements.

Oliver-BE commented:
> I have fixed padding and NICFI button. We don't need to sweat the details to be pixel perfect comparing to previous one. We will style it all over again using Andrea designs.
> 
> The other css files `like bootstrap-grid` and `tabulator` were not being served but just sitting in `resources` directory, I have removed them. `gmw_global.css` apply styles to html elements (reseting the styles) and `gmw_main.css` defines style classes that get used by those html elements.

Thanks for fixing those things. Yeah, that makes sense. I was just pointing out a few obvious places where it changed but we will very soon be overriding all of the styles. Thanks for the info on the CSS files. Also, I think I figured out that `gmw` stands for "Gold Mine Watch". Perhaps we can update `gmw_global.css` and `gmw_main.css` to `comimo_global.css` and `comimo_main.css`? Also, we should figure out if we are still going to use the `_main.css` file for style classes with the new UI that Andrea is proposing.
